config/deployment/prod/dispatch.yaml

```diff
@@ -13,3 +13,7 @@ dispatch:
     service: pepper-backend
   - url: "pepper.datadonationplatform.org/*"
     service: pepper-backend
+  - url: "dsm.datadonationplatform.org/api/*"
```

Comment:
These dispatch files can be safely deployed even while DSM is on docker.  These won't take effect until the app engine instance is created and custom domains are configured.